### ISSUE TITLE
How to train my own datasets? Thanks a lot.

### ISSUE THREAD

**Issue by WWaynee** [prasunroy/rcnnpose-pytorch#1]

**Comment by prasunroy** [comment]
Hi,
The original intention of this repository was only to provide a higher level api wrapper on the official PyTorch implementations of Mask R-CNN and Keypoint R-CNN for human pose estimation. So, it doesn't come with training and testing scripts. For your purpose i.e. training on a custom dataset, please check the official PyTorch tutorials on [1] `Object Detection Finetuning` and [2] `Transfer Learning`.

[1] https://pytorch.org/tutorials/intermediate/torchvision_tutorial.html
[2] https://pytorch.org/tutorials/beginner/transfer_learning_tutorial.html

**Comment by WWaynee** [comment]
收到了惹，啾咪~
